docs/update-design-question.md

```diff
@@ -0,0 +1,45 @@
+Guys, we have a question for Formica where everyone is invited to participate deciding.
+
+So, formica currently is a tool for managing groups of units. Example:
+
+```
+$ formicactl submit elasticsearch@{1,2}
+$ formciactl start elasticsearch@{1,2}
+$ formicactl status elasticsearch
+Group           Unit  <more headers>
+elasticsearch@1 *     active  active  running 10.0.4.100
+elasticsearch@2 *     active  active  running 10.0.4.101
+```
+
+As you see, it is very similar to `fleetctl`. Next we want to implement an update
+functionality, similar (but better ;) than what `releaseit` currently does.
+
+## Question 1
+In the example above we have used `1` and `2` for the slices. When updating, we
+want the users to have the option to create new instances first. This raises where
+the names for those slices come from.
+
+* Option 1: Just increment the found ids. So if we find `7,8,9` we continue with
```

Comment:
Option1: I would find it a bit awkard whenever I start to remove/add units.. Moreover I'd like the units to be shown by age (older/newer).

Option2: Random ids is the way to go in my opinion. In fact, the group should be elasticsearch(pod) and its ID elasticsearch@123131. If you wanna stop a pod-group you just use group names plus the ID. elasticsearch@RANDOM_ID. I know that this differs with what we have but it'd be closer to the way this kind of things are managed by other tools, common cli's, or kubelet, rkt,